### ISSUE TITLE
Android crashes on recognizeTextCloud

### ISSUE THREAD

**Issue by rdlauer** [EddyVerbruggen/nativescript-plugin-firebase#878]
Just upgraded an app to 6.5.0. iOS works fine, but on Android the app is crashing when I execute `firebase.mlkit.textrecognition.recognizeTextCloud`.

Anything related to text recognition that changed with 6.5.0 that may be the culprit? Device is Android 7.

Unfortunately the console log isn't providing much help:

```
08-27 14:27:17.151 23313 23313 F art     : art/runtime/java_vm_ext.cc:470]     from java.lang.Object com.tns.Runtime.callJSMethodNative(int, int, java.lang.String, int, boolean, java.lang.Object[])
08-27 14:27:17.151 23313 23313 F art     : art/runtime/java_vm_ext.cc:470]   at com.tns.Runtime.callJSMethodNative(Native method)
08-27 14:27:17.151 23313 23313 F art     : art/runtime/java_vm_ext.cc:470]   at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1116)
08-27 14:27:17.151 23313 23313 F art     : art/runtime/java_vm_ext.cc:470]   at com.tns.Runtime.callJSMethodImpl(Runtime.java:996)
08-27 14:27:17.151 23313 23313 F art     : art/runtime/java_vm_ext.cc:470]   at com.tns.Runtime.callJSMethod(Runtime.java:983)
08-27 14:27:17.151 23313 23313 F art     : art/runtime/java_vm_ext.cc:470]   at com.tns.Runtime.callJSMethod(Runtime.java:967)
08-27 14:27:17.151 23313 23313 F art     : art/runtime/java_vm_ext.cc:470]   at com.tns.Runtime.callJSMethod(Runtime.java:959)
08-27 14:27:17.307 23313 23313 F art     : art/runtime/runtime.cc:441]   at com.tns.Runtime.callJSMethodNative(Native method)
08-27 14:27:17.307 23313 23313 F art     : art/runtime/runtime.cc:441]   at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1116)
08-27 14:27:17.307 23313 23313 F art     : art/runtime/runtime.cc:441]   at com.tns.Runtime.callJSMethodImpl(Runtime.java:996)
08-27 14:27:17.307 23313 23313 F art     : art/runtime/runtime.cc:441]   at com.tns.Runtime.callJSMethod(Runtime.java:983)
08-27 14:27:17.307 23313 23313 F art     : art/runtime/runtime.cc:441]   at com.tns.Runtime.callJSMethod(Runtime.java:967)
08-27 14:27:17.307 23313 23313 F art     : art/runtime/runtime.cc:441]   at com.tns.Runtime.callJSMethod(Runtime.java:959)
08-27 14:27:17.308 23313 23313 F art     : art/runtime/runtime.cc:441]   at com.tns.Runtime.callJSMethodNative(Native method)
08-27 14:27:17.308 23313 23313 F art     : art/runtime/runtime.cc:441]   at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1116)
08-27 14:27:17.308 23313 23313 F art     : art/runtime/runtime.cc:441]   at com.tns.Runtime.callJSMethodImpl(Runtime.java:996)
08-27 14:27:17.308 23313 23313 F art     : art/runtime/runtime.cc:441]   at com.tns.Runtime.callJSMethod(Runtime.java:983)
08-27 14:27:17.308 23313 23313 F art     : art/runtime/runtime.cc:441]   at com.tns.Runtime.callJSMethod(Runtime.java:967)
08-27 14:27:17.308 23313 23313 F art     : art/runtime/runtime.cc:441]   at com.tns.Runtime.callJSMethod(Runtime.java:959)
```

**Comment by EddyVerbruggen** [comment]
Hmm, there was a major refactor in their SDK (see #853) but I thought I tested all of it. Will take a look tomorrow.

**Comment by EddyVerbruggen** [comment]
Strangely the code I used was according to [their sample]https://firebase.google.com/docs/ml-kit/android/recognize-text). Aaaaanyway, this is now fixed and will be part of 6.6.0 which I'll try to release today.
